openapi.yaml

```diff
@@ -115,7 +115,7 @@ paths:
           $ref: '#/components/responses/409'
         '500':
           $ref: '#/components/responses/500'
-  /books/{id}:
+  /library/books/{id}:
```

Comment:
As discussed, `id` should be something like `bookId`. Same thing for other ids.

Reply:
done.